lab-anthony/route/drink-router.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const Router = require('express').Router;
+const drinkRouter = module.exports = new Router();
+const debug = require('debug')('drink:drink-router');
+const storage = require('../lib/storage');
+const Drink = require('../model/drink');
+const bodyParser = require('body-parser').json();
+
+function createDrink(reqBody){
+  debug('createDrink');
+  return new Promise(function(resolve, reject){
+    var drink;
+    try{
+      drink = new Drink(reqBody.drinkName, reqBody.cafeName);
+    } catch(err) {
+      reject(err);
+    }
+    storage.setItem('drink', drink).then(function(drink){
```

Comment:
This is a style thing, but it's nice if you chain functions like this to keep indentation consistent: 

```javascript
storage.setItem('drink', drink)
  .then(function(drink){
  // blah blah
  })
  .catch(function(err){
  // blah blah
  })
```